src/Runner.jl

```diff
@@ -1288,7 +1290,7 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
             mapping["GNU_LIBC_VERSION"] = "glibc 2.12.2"
         elseif arch(platform) in ("armv7l", "aarch64")
             mapping["GNU_LIBC_VERSION"] = "glibc 2.19"
-        elseif arch(platform) === "powerpc64le"
+        elseif arch(platform) in ("powerpc64le", "riscv64")
```

Comment:
I'm not really sure this makes sense: [glibc 2.17 is from December 2012](https://sourceware.org/glibc/wiki/Glibc%20Timeline), [RISC-V was introduced in 2014](https://www2.eecs.berkeley.edu/Pubs/TechRpts/2014/EECS-2014-146.pdf), we need to find a version of glibc which actually supports RISC-V.

Reply:
Using 2.35 now.